tests/src/kernel/hipLanguageExtensions.cpp

```diff
@@ -28,10 +28,11 @@ THE SOFTWARE.
  */
 
 #include "hip/hip_runtime.h"
+
 #include <test_common.h>
 
 #ifdef __HCC__
-#include <amp.h>
+    #include <hc.hpp>
```

Comment:
This change has already been submitted and merged since HCC broke this HIP test. So we can discard this change from this PR.